CMakeLists.txt

```diff
@@ -475,7 +485,19 @@ else ()
   set(caf_dir ${CAF_INCLUDE_DIR_CORE})
 endif ()
 
+# Figure out whether we point to a build directory or a prefix.
```

Comment:
> Only for the summary output?

Yes.

> Why not simply print BROKER_LIBRARY?

Consistency: all other status reports show the include directory (or prefix).

Reply:
I don't think this is consistent. Some summaries consider the include path (snappy, pcap) while others consider the library path (CAF, Broker). I would prefer to simply print the actual variable name and its content instead to clearly communicate to the user what the results of the various `FindX` calls were.

If you insist on this form of summary, then we should at least turn the repeated code into a macro.

Reply:
> I don't think this is consistent. Some summaries consider the include path (snappy, pcap) while others consider the library path (CAF, Broker)

CAF and Broker show the prefix (e.g., `/usr/local`) when it is found there and show the *build* directory otherwise (because there might be multiple build directories per source tree). CAF and Broker are the exception here, because only those support the point-to-build-directory feature. All others show the prefix.

So for true consistency, we could show the libraries instead. But this would break for header-only dependencies.

(All of this should go in a separate PR though.)

Reply:
> (All of this should go in a separate PR though.)

Yeah, let's continue this discussion elsewhere.